src/styles/CompanyDetails.module.css

```diff
@@ -0,0 +1,45 @@
+.container {
```

Comment:
- [ ] OPTIONAL
The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Also Two cells should be displayed on the mobile screen as seen the [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the cells/ card of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/182989713-58e42501-b6b3-4c32-9c42-71dab8254a20.png)

**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)

Reply:
@rloterh could you please explain it in different way because I didn't understand?

Reply:
@MohammedElseddik 
For each row of double cells, one cell should be of a different shade or slight background color change from the other. So the cells below and at the side of a particular color should be different from the cell.

**An example of this pattern is as below:**
![image](https://user-images.githubusercontent.com/12745474/182995158-0f7713b5-7256-4fd3-b1ee-80debcb857c7.png)

Cheers! 😃